src/ucp/api/ucp.h

```diff
@@ -639,6 +678,33 @@ void ucp_context_print_info(ucp_context_h context, FILE *stream);
 ucs_status_t ucp_worker_create(ucp_context_h context, ucs_thread_mode_t thread_mode,
                                ucp_worker_h *worker_p);
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief Create a worker object with list of parameters.
+ *
+ * This routine allocates and initializes a @ref ucp_worker_h "worker" object.
+ * Each worker is associated with one and only one @ref ucp_context_h
+ * "application" context.  In the same time, an application context can create
+ * multiple @ref ucp_worker_h "workers" in order to enable concurrent access to
+ * communication resources. For example, application can allocate a dedicated
+ * worker for each application thread, where every worker can be progressed
+ * independently of others.
+ *
+ * @note The worker object is allocated within context of the calling thread
```

Comment:
@yosefe May I propose a counter proposal? From the user point of view, It would be good to have ucp_worker_create() with default parameters in UCP interface. It will be the good example how to initialize parameters (need to move into ucp.h) and save the user time if the user need simple ucp_worker_create().
Proposal is:
- In `ucp_worker_create()` documentation, write that this is the same as `ucp_worker_create_param()` with default parameters.
- Move definition of the ucp_worker_create() from `.c` to `.h` to show the body of the function to the user (as example) and to the compiler (avoid extra jmp and expect inlining).
- If the user doesn't need parametrized function, ucp_worker_create() will be a good candidate.